bin/riemann-elasticsearch

```diff
@@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
   end
 
   def health_url
-    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
+    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
```

Comment:
Ah, that too--I actually meant that by default, this is gonna hit `//_cluster/health`, not `/_cluster/health`.